src/com/google/enterprise/adaptor/filenet/FileNetAdaptor.java

```diff
@@ -83,6 +110,8 @@ public void init(AdaptorContext context) throws Exception {
       throw new StartupException(
           "Failed to access content engine's object store", e);
     }
+
+    documentTraverser = new MockTraverser(maxFeedUrls);
```

Comment:
If I'm understanding your requirement for 2, don't you want maxFeedUrls - 1 here?